docs/element-definitions.md

```diff
@@ -107,12 +105,15 @@ Arbitrary data asset.
 
 - `element`: asset (string, fixed)
 - `meta`
-    - `classes` (array, fixed)
-        - (enum[string])
-            - messageBody - Asset is an example of message-body
-            - messageBodySchema - Asset is an schema for message-body
-- `attributes` (object)
-    - `contentType` (string) - Optional media type of the asset. When this is unset, the content type SHOULD be inherited from the `Content-Type` header of a parent HTTP Message Payload
+    - `classes` (Array Element)
+        - `content` (array, fixed-type)
+            - One Of
```

Comment:
They are both String, but they have different fixed content values. @w-vi do you have another suggestion on how to format this?